projects/storefrontlib/src/cms-components/product/stock-notification/stock-notification-dialog/stock-notification-dialog.component.html

```diff
@@ -2,7 +2,13 @@
   <div class="cx-dialog-title modal-title">
     {{ 'stockNotification.subscriptionDialog.header' | cxTranslate }}
   </div>
-  <button type="button" class="close" aria-label="Close" (click)="close()">
+  <button
+    type="button"
+    class="close"
+    aria-label="Close"
+    tabindex="-1"
```

Comment:
Should this button be excluded from tabbing?

Reply:
Because this button is placed on top-right corner and has the same function(close dialog) with another button, so we disabled it from tabbing.